source/installing-splunk/splunk-basic.rst

```diff
@@ -47,7 +47,23 @@ This component works receiving the data flow streamed by a forwarder and stores
 
       # dpkg --install splunk-enterprise-package.deb
 
-3. Ensure Splunk v7.3.0 is installed in ``/opt/splunk`` and start the service:
+3. Configure ``inputs.conf`` and ``indexes.conf``
+
+ Splunk needs the `indexes.conf` file to create Wazuh indexes and the `inputs.conf` file to listen to forwarded data on port 9997. To create these file please execute the following commands:
```

Comment:
Grammar typo:
- `To create these file` -> `to create these files`

Reply:
Solved here: https://github.com/wazuh/wazuh-documentation/pull/1271/commits/f4d0a82b3dc89b087956b92a641133447b276708